tools/bisection/run-dict-surface-diff.sh

```
@@ -0,0 +1,115 @@
+# 3 = a probe family is inactive.
+#
+# Env:
+#   PINYIN_ORACLE_PREFIX  pin-built oracle prefix
+#                         (default ~/.local/opt/pinyin-oracle)
+#   DICT_SYSTEM           real-unigram capi system dir (5 redb/text
+#                         files). When unset, one is assembled into
+#                         target/datagen/redb from the pinned model20
+#                         archive (fetch-model.sh + oxpinyin-datagen).
+
+set -u
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)